src/app.py

```
@@ -25,16 +27,25 @@
+    # start_time = time.time()
     # ---
     if event is None:
         return jsonify({"error": "حدث خطأ أثناء تحميل المكتبة"})
     # ---
     json_result = event([title], tst_prnt_all=False) or {"result": ""}
     # ---
+    # delta = time.time() - start_time
+    # ---
+    data = {}
+    # ---
     for x, v in json_result.items():
```

Ruff (0.8.2)

Loop control variable `x` not used within loop body

Rename unused `x` to `_x`

(B007)